packages/admin/src/emailCampaigns/form/EmailCampaignForm.tsx

```diff
@@ -227,77 +230,85 @@ export function EmailCampaignForm({ id, EmailCampaignContentBlock, scope, previe
-                                <AdminTabLabel>
-                                    <FormattedMessage id="cometBrevoModule.emailCampaigns.blocks" defaultMessage="Blocks" />
-                                </AdminTabLabel>
-                            ),
-                            content: (
-                                <BlocksFinalForm
-                                    onSubmit={(values) => setState({ ...state, ...values })}
-                                    initialValues={{
-                                        content: state?.content,
-                                    }}
-                                >
-                                    <AdminComponentRoot>
-                                        <Field name="content" fullWidth required component={FinalFormEmailCampaignContentBlock} />
-                                    </AdminComponentRoot>
-                                </BlocksFinalForm>
-                            ),
-                        },
-                        {
-                            key: "send-manager",
-                            label: (
-                                <AdminTabLabel>
-                                    <FormattedMessage id="cometBrevoModule.emailCampaigns.sendManager" defaultMessage="Send manager" />
-                                </AdminTabLabel>
-                            ),
-                            content: (
-                                <BlocksFinalForm
-                                    onSubmit={(values) => setState({ ...state, scheduledAt: values.scheduledAt, targetGroup: values.targetGroup })}
-                                    initialValues={{
-                                        targetGroup: state.targetGroup,
-                                        scheduledAt: state.scheduledAt,
-                                    }}
-                                >
-                                    <SendManagerFields
-                                        scope={scope}
-                                        disableScheduling={isScheduleDateDisabled}
-                                        isSendable={!hasChanges && state.targetGroup != undefined}
-                                        id={id}
-                                    />
-                                    <TestEmailCampaignForm id={id} isSendable={!hasChanges && state.targetGroup != undefined} />
-                                </BlocksFinalForm>
-                            ),
-                        },
-                    ]}
-                </BlockPreviewWithTabs>
+                {isViewOnly ? (
+                    <IFrameBridgeProvider key={previewUrl}>
+                        <BlockPreview url={previewUrl} previewState={previewState} previewApi={previewApi} />
+                    </IFrameBridgeProvider>
+                ) : (
+                    <BlockPreviewWithTabs previewUrl={previewUrl} previewState={previewState} previewApi={previewApi}>
```

Comment:
Optional:
I think it might be better to split the component into two components, because with a few more functions this EmailCampaignForm will become quite complex, but it would be okay for me to leave it as it is in the first step.

Reply:
6fb2f17bc8f4e132c8d42ebb0422e6e4f2e2b4ce